doc/effects.md

```diff
@@ -20,10 +20,23 @@ If `isInfinite` is false and `isAlternating` is true the effect will go from the
 
 When an effect is completed the callback `onComplete` will be called, it can be set as an optional argument to your effect.
 
+## Common for MoveEffect, ScaleEffect and RotateEffect (SimplePositionComponentEffects)
+A common thing for `MoveEffect`, `ScaleEffect` and `RotateEffect` is that it takes `duration` and `speed` as arguments.
+
+- Duration means the time it takes for one iteration from beginning to end, with alternation taken into account (but not `isInfinite`).
```

Comment:
can you use duration if `isInfinite`? maybe you can rephrase to make it clear that if isInfinite is set, you can only use speed.

Reply:
You can still use it, you set the length of one iteration if the effect is infinite.
That is what I'm trying to say in this sentence, maybe it was not very clearly formulated.

Reply:
I got it now, lg. Maybe add an example of this particular case? as it is the more non-obvious one

Reply:
Added to the docs